### ISSUE TITLE
E-Values

### ISSUE THREAD

**Issue by Drob84761** [soedinglab/MMseqs2#721]
What E-Values and sequence identity should I be shooting for in an easy-search workflow? I'm having a hard time to understand what reduces false positives the most.

**Comment by milot-mirdita** [comment]
I don't think you need much changing. The defaults are set to optimize sensitivity while keeping false positives low.

You can further decrease the E-value threshold, but you can do that also after the search just based on the result file.

Sequence identity, coverage etc. depend a lot on your specific research question.

**Comment by martin-steinegger** [comment]
Closing this one due to inactivity — it was answered above and we haven't heard back. If it's still relevant, please reopen with your MMseqs2 version, the exact command, and the full output, and we'll be happy to help. Thanks!
